source/manual/manage-assets.html.md.erb

```diff
@@ -8,28 +8,14 @@ parent: "/manual.html"
 
 ## Get an asset's ID
 
-You can get an asset's ID by using it's URL:
+If the asset URL starts with `/media`, the asset ID is in the URL.
+For example, for the URL `https://assets.publishing.service.gov.uk/media/65f2c1110e1c2f8c4dffaa53/my_file.jpg`, the ID is `65f2c1110e1c2f8c4dffaa53`.
 
-1. Open a Rails console:
+For assets that do not start `/media`, obtain the ID by running the following rake task:
 
-    ```sh
-    k exec -it deploy/asset-manager -- rails c
-    ```
+<%= RunRakeTask.links("asset-manager", "assets:get_id_by_legacy_url_path[SLUG]") %>
```

Comment:
Should we add quotation marks/apostrophes around the argument, as [...] is interpreted by the shell?

Reply:
I don't think it would hurt, but it's also not necessary - e.g. I ran the `assets:redirect` rake task on production earlier and without quotes.

Reply:
I did run the line and the shell complained.

```
zsh: no matches found: assets:get_id_by_legacy_url_path[/government/uploads/uploaded/hmrc/realtimepayetools-update-v23.xml]
```

Reply:
Thanks for checking - I've added quotes.